### DISCUSSION TITLE
News Panel: Add Ability to Restrict Description Length

### DISCUSSION THREAD

**Discussion by NotAFile** [grafana/grafana#37020] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
The ability to ellipsize the article description in the news panel after some length e.g. in lines.

**Why is this needed**:
Many RSS feeds put a lot of text into the description. Since the news panel strips formatting such as newlines, this ends up as a huge illegible blob of text that hides other content.

**Comment by rakeshraih** [comment]
Can I take this one?

**Comment by pkolyvas** [comment]
Issue is still up for grabs!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
